pwa/src/components/objectEntities/ObjectEntityFormNew.tsx

```diff
@@ -9,11 +9,11 @@ import LoadingOverlay from "../loadingOverlay/loadingOverlay";
 import { Link } from 'gatsby';
 
 interface ObjectEntityFormNewProps {
-  objectEntityId: string,
+  objectId: string,
   entityId: string,
 }
 
-export const ObjectEntityFormNew: React.FC<ObjectEntityFormNewProps> = ({ objectEntityId, entityId }) => {
+export const ObjectEntityFormNew: React.FC<ObjectEntityFormNewProps> = ({ objectId, entityId }) => {
```

Comment:
Why is there a `ObjectEntityFormNew`? I see that the `ObjectEntityForm` still exists in our codebase but is never implemented.

Could we just delete the old one and call this one `ObjectEntityForm`? Instead of cluttering our codebase. 

Maybe you can (1) check  (I think with @bbrands02) if I'm correct and fix it immediately or (2) create a JIRA ticket.

Reply:
I spoke with @bbrands02 and the file wil be used and renamed. That wil be done under an already existing JIRA ticket.